### PR TITLE
Ds-745-Select Fixes

### PR DESCRIPTION
**Pull request by zuzannanap**

### CODE REVIEW

File: packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx [hunk 1]
```diff
@@ -213,6 +212,7 @@ export const SelectDropdownComponents = {
     );
   },
   Input: ({ id, ...props }: InputProps<IDropdownOption>) => {
+    //@ts-ignore
```
Comment (monikaracka) [inline packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx:215]: Why did you add this here?  I don't see here any problem whit types.

**Review by monikaracka — APPROVED** [review]

File: packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx (hunk 1 above)
Comment (zuzannanap) [reply packages/design-system/src/components/Dropdown/SelectDropdownComponents.tsx:215]: it started to throw error in my console for some reason
